### PR TITLE
Game stats dc

### PR DESCRIPTION
**Pull request by dcoleman21**
Ran the spec_harness and made changes to #percentage_ties and #average_goals_per_game tests, methods on both stat_tracker and game_stats to reflect the accurate return values.

### CODE REVIEW

**Review by araspaas — APPROVED** [review]
changes look good.

File: lib/game_stats.rb [hunk 1]
```diff
@@ -48,9 +48,9 @@ def count_of_games_by_season
 
   def average_goals_per_game
     gpg = @game_teams.sum do |game|
-      game.goals.to_i
+      game.goals.to_f
```
Comment (ShaunDaneJames) [inline lib/game_stats.rb:51]: Good catch!

**Comment by robertorodriguez12** [comment]
Like the changes!!!

**Review by ShaunDaneJames — APPROVED** [review]
Nice find Dani!

**Review by robertorodriguez12 — APPROVED** [review]

**Pull request merged**